services/authService.js

```diff
@@ -0,0 +1,30 @@
+import bcrypt from 'bcrypt';
+import jwt from "jsonwebtoken";
+import 'dotenv/config'
+
+
+const SECRET_KEY = process.env.SECRET_KEY; // move  this to gh secrets or something
+
+export class AuthService{
+  async hashPassword(password){
+    const hashedPassword = await bcrypt.hash(password, 10);
+    return hashedPassword
+  }
+  generateToken(username) {
+    const token = jwt.sign({ username: username }, SECRET_KEY, {
+      expiresIn: '1h'
```

Comment:
Given that you have no refresh tokens, I suggest you to increase this time for now